gtsam/nonlinear/Values-inl.h

```diff
@@ -294,11 +294,12 @@ namespace gtsam {
    // Handle dynamic matrices
    template <int M, int N>
    struct handle_matrix<Eigen::Matrix<double, M, N>, true> {
-     Eigen::Matrix<double, M, N> operator()(Key j, const Value* const pointer) {
-       try {
+     inline Eigen::Matrix<double, M, N> operator()(Key j, const Value* const pointer) {
```

Comment:
Since we're fixing this, one more `try catch` above (in `struct handle`) this functor should be done the same way...